[2024-11-04T00:00:03.991+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-04T00:00:04.006+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: self-tutorial-dag.echo_input scheduled__2024-11-03T00:00:00+00:00 [queued]>
[2024-11-04T00:00:04.014+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: self-tutorial-dag.echo_input scheduled__2024-11-03T00:00:00+00:00 [queued]>
[2024-11-04T00:00:04.015+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-04T00:00:04.026+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): echo_input> on 2024-11-03 00:00:00+00:00
[2024-11-04T00:00:04.031+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1179) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-04T00:00:04.032+0000] {standard_task_runner.py:72} INFO - Started process 1182 to run task
[2024-11-04T00:00:04.033+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'self-tutorial-dag', 'echo_input', 'scheduled__2024-11-03T00:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/self-tutorial-dag.py', '--cfg-path', '/tmp/tmp3ghp4mxb']
[2024-11-04T00:00:04.034+0000] {standard_task_runner.py:105} INFO - Job 121: Subtask echo_input
[2024-11-04T00:00:04.070+0000] {task_command.py:467} INFO - Running <TaskInstance: self-tutorial-dag.echo_input scheduled__2024-11-03T00:00:00+00:00 [running]> on host 2839f63d218a
[2024-11-04T00:00:04.139+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='self-tutorial-dag' AIRFLOW_CTX_TASK_ID='echo_input' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-03T00:00:00+00:00'
[2024-11-04T00:00:04.140+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-04T00:00:04.161+0000] {logging_mixin.py:190} INFO - ##################################################About to fall a sleep
[2024-11-04T00:00:04.161+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-04T00:00:04.170+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-04T00:00:04.170+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=self-tutorial-dag, task_id=echo_input, run_id=scheduled__2024-11-03T00:00:00+00:00, execution_date=20241103T000000, start_date=20241104T000004, end_date=20241104T000004
[2024-11-04T00:00:04.207+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-04T00:00:04.218+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-04T00:00:04.221+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
